acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/ThreadPantheonNodeRunner.java

```diff
@@ -69,6 +70,7 @@ public void startNode(final PantheonNode node) {
               .devMode(node.isDevMode())
               .nodePrivateKeyFile(KeyPairUtil.getDefaultKeyFile(node.homeDirectory()))
               .metricsSystem(noOpMetricsSystem)
+              .maxPendingTransactions(PendingTransactions.MAX_PENDING_TRANSACTIONS)
```

Comment:
`PantheonControllerBuilder` has been updated to set `PendingTransactions.MAX_PENDING_TRANSACTIONS` as default when the parameter hasn't been set. We can remove this line.